### ISSUE TITLE
Failed to install chaincodes, Error: Failed to send install Proposal or receive valid response

### ISSUE THREAD

**Issue by lotty02cho** [hyperledger/caliper#166]
I try to use zookeeper using simpletest.go

So, I try to open another terminal and run below commands.

`cd /opt/gopath/src/github.com/hyperledger/caliper/`
`npm run startclient -- xxx.xxx.xx.xx:2181`

next, I try to run caliper.
`node benchmark/simple/main.js -c config.json -n fabric.json`

however, the issues comes out.

`hyper@hyperledger001:/opt/gopath/src/github.com/hyperledger/caliper$ node benchmark/simple/main.js -c config.json -n fabric.json
TAP version 13
# #######Caliper Test######
# Sleep 5s......
# Join channel......
# install all chaincodes......
info: [packager/Golang.js]: packaging GOLANG from contract/fabric/simple/go
info: [packager/Golang.js]: packaging GOLANG from contract/fabric/simple/go
error: [E2E testing]: install proposal was bad
error: [E2E testing]: install proposal was bad
not ok 1 Failed to install chaincodes, Error: Failed to send install Proposal or receive valid response: [object Object],[object Object] at Client.newDefaultKeyValueStore.then.then.then (/opt/gopath/src/github.com/hyperledger/caliper/src/fabric/e2eUtils.js:144:19) at <anonymous>
  ---
    operator: fail
    at: chaincodes.reduce.then.catch (/opt/gopath/src/github.com/hyperledger/caliper/src/fabric/install-chaincode.js:65:19)
    stack: |-
      Error: Failed to install chaincodes, Error: Failed to send install Proposal or receive valid response: [object Object],[object Object]
          at Client.newDefaultKeyValueStore.then.then.then (/opt/gopath/src/github.com/hyperledger/caliper/src/fabric/e2eUtils.js:144:19)
          at <anonymous>
          at Test.assert [as _assert] (/opt/gopath/src/github.com/hyperledger/caliper/node_modules/tape/lib/test.js:224:54)
          at Test.bound [as _assert] (/opt/gopath/src/github.com/hyperledger/caliper/node_modules/tape/lib/test.js:76:32)
          at Test.fail (/opt/gopath/src/github.com/hyperledger/caliper/node_modules/tape/lib/test.js:317:10)
          at Test.bound [as fail] (/opt/gopath/src/github.com/hyperledger/caliper/node_modules/tape/lib/test.js:76:32)
          at chaincodes.reduce.then.catch (/opt/gopath/src/github.com/hyperledger/caliper/src/fabric/install-chaincode.js:65:19)
          at <anonymous>
  ...
fabric.installSmartContract() failed, Error: Failed to send install Proposal or receive valid response: [object Object],[object Object]
    at Client.newDefaultKeyValueStore.then.then.then (/opt/gopath/src/github.com/hyperledger/caliper/src/fabric/e2eUtils.js:144:19)
    at <anonymous>
[Transaction Info] - Submitted: 0 Succ: 0 Fail:0 Unfinished:0
unexpected error, Error: Failed to send install Proposal or receive valid response: [object Object],[object Object]
    at Client.newDefaultKeyValueStore.then.then.then (/opt/gopath/src/github.com/hyperledger/caliper/src/fabric/e2eUtils.js:144:19)
    at <anonymous>
error: [E2E testing]: install proposal was bad
error: [E2E testing]: install proposal was bad

1..1
# tests 1
# pass  0
# fail  1
`

So, I entered my peer container and check my logs.
`docker container logs peer0.org1.kisa.com`

`2018-08-24 08:53:29.186 UTC [chaincode] ProcessStream -> ERRO bba handling chaincode support stream: rpc error: code = Canceled desc = context canceled
receive failed
github.com/hyperledger/fabric/core/chaincode.(*Handler).ProcessStream
        /opt/gopath/src/github.com/hyperledger/fabric/core/chaincode/handler.go:383
github.com/hyperledger/fabric/core/chaincode.(*ChaincodeSupport).HandleChaincodeStream
        /opt/gopath/src/github.com/hyperledger/fabric/core/chaincode/chaincode_support.go:154
github.com/hyperledger/fabric/core/chaincode.(*ChaincodeSupport).Register
        /opt/gopath/src/github.com/hyperledger/fabric/core/chaincode/chaincode_support.go:159
github.com/hyperledger/fabric/core/chaincode/accesscontrol.(*interceptor).Register
        /opt/gopath/src/github.com/hyperledger/fabric/core/chaincode/accesscontrol/interceptor.go:57
github.com/hyperledger/fabric/protos/peer._ChaincodeSupport_Register_Handler
        /opt/gopath/src/github.com/hyperledger/fabric/protos/peer/chaincode_shim.pb.go:472
github.com/hyperledger/fabric/vendor/google.golang.org/grpc.(*Server).processStreamingRPC
        /opt/gopath/src/github.com/hyperledger/fabric/vendor/google.golang.org/grpc/server.go:1059
github.com/hyperledger/fabric/vendor/google.golang.org/grpc.(*Server).handleStream
        /opt/gopath/src/github.com/hyperledger/fabric/vendor/google.golang.org/grpc/server.go:1152
github.com/hyperledger/fabric/vendor/google.golang.org/grpc.(*Server).serveStreams.func1.1
        /opt/gopath/src/github.com/hyperledger/fabric/vendor/google.golang.org/grpc/server.go:637
runtime.goexit
        /opt/go/src/runtime/asm_amd64.s:2361
2018-08-24 09:01:55.010 UTC [discovery/lifecycle] func1 -> ERRO bbb Chaincode deploy for simple failed
2018-08-24 09:01:55.010 UTC [endorser] ProcessProposal -> ERRO bbc [][431331c1] simulateProposal() resulted in chaincode name:"lscc"  response status 500 for txid: 431331c1e56047032ff7c028f5f236ecf32237c5d21e70ad5e5fa0fa2b18a5a3
2018-08-24 09:08:43.756 UTC [discovery/lifecycle] func1 -> ERRO bbd Chaincode deploy for simple failed
2018-08-24 09:08:43.756 UTC [endorser] ProcessProposal -> ERRO bbe [][45d22163] simulateProposal() resulted in chaincode name:"lscc"  response status 500 for txid: 45d22163d6a7af12bb372d948d00f388c1aec5fcf9812030dd2adb5f696a36d7
hyper@hyperledger001:/opt/gopath/src/github.com/hyperledger/caliper$ docker container logs peer0.org1.kisa.comdocker container logs peer0.org1.kisa.com
`

while I have cloned and edited this project, I never changed golang chaincode,
but I changed local to zookeeper in config.json like below.
`    "clients": {
      "type": "zookeeper",
      "zoo" : {
        "server": "172.27.42.57:2181",
        "clientsPerHost": 5
      }
    },
`
Have you guys saw about this issues?

**Comment by lotty02cho** [comment]
I think it cause chaincode version. I changed my chaincode version in fabric.json, the error is solved, but when caliper try to instantiate it, error appears like this.

`hyper@hyperledger001:/opt/gopath/src/github.com/hyperledger/caliper$ node benchmark/simple/main.js -c config.json -n fabric.json
TAP version 13
# #######Caliper Test######
# Sleep 5s......
# Join channel......
# install all chaincodes......
info: [packager/Golang.js]: packaging GOLANG from contract/fabric/simple/go
info: [packager/Golang.js]: packaging GOLANG from contract/fabric/simple/go
ok 1 Installed chaincode simple successfully in all peers
# Instantiate chaincode......
not ok 2 Failed to instantiate chaincodes, Error: Failed to send instantiate due to error: Error: Failed to send instantiate Proposal or receive valid response. Response null or status is not 200. exiting... at Client.newDefaultKeyValueStore.then.then.then.then (/opt/gopath/src/github.com/hyperledger/caliper/src/fabric/e2eUtils.js:372:19) at <anonymous> at Client.newDefaultKeyValueStore.then.then.then.then.then (/opt/gopath/src/github.com/hyperledger/caliper/src/fabric/e2eUtils.js:385:15) at <anonymous>
  ---
    operator: fail
    at: chaincodes.reduce.then.catch (/opt/gopath/src/github.com/hyperledger/caliper/src/fabric/instantiate-chaincode.js:60:19)
    stack: |-
      Error: Failed to instantiate chaincodes, Error: Failed to send instantiate due to error: Error: Failed to send instantiate Proposal or receive valid response. Response null or status is not 200. exiting...
          at Client.newDefaultKeyValueStore.then.then.then.then (/opt/gopath/src/github.com/hyperledger/caliper/src/fabric/e2eUtils.js:372:19)
          at <anonymous>
          at Client.newDefaultKeyValueStore.then.then.then.then.then (/opt/gopath/src/github.com/hyperledger/caliper/src/fabric/e2eUtils.js:385:15)
          at <anonymous>
          at Test.assert [as _assert] (/opt/gopath/src/github.com/hyperledger/caliper/node_modules/tape/lib/test.js:224:54)
          at Test.bound [as _assert] (/opt/gopath/src/github.com/hyperledger/caliper/node_modules/tape/lib/test.js:76:32)
          at Test.fail (/opt/gopath/src/github.com/hyperledger/caliper/node_modules/tape/lib/test.js:317:10)
          at Test.bound [as fail] (/opt/gopath/src/github.com/hyperledger/caliper/node_modules/tape/lib/test.js:76:32)
          at chaincodes.reduce.then.catch (/opt/gopath/src/github.com/hyperledger/caliper/src/fabric/instantiate-chaincode.js:60:19)
          at <anonymous>
  ...
fabric.installSmartContract() failed, Error: Fabric: instantiate chaincodes failed
    at chaincodes.reduce.then.catch (/opt/gopath/src/github.com/hyperledger/caliper/src/fabric/instantiate-chaincode.js:61:31)
    at <anonymous>
[Transaction Info] - Submitted: 0 Succ: 0 Fail:0 Unfinished:0
unexpected error, Error: Fabric: instantiate chaincodes failed
    at chaincodes.reduce.then.catch (/opt/gopath/src/github.com/hyperledger/caliper/src/fabric/instantiate-chaincode.js:61:31)
    at <anonymous>

1..2
# tests 2
# pass  1
# fail  1
`

I saw my docker container by using cli, Installing chaincode is fine, but instantiated not completed.

**Comment by lotty02cho** [comment]
I think it cause chaincode version. I changed my chaincode version in fabric.json, the error is solved, but when caliper try to instantiate it, error appears like this.

**Comment by lotty02cho** [comment]
however, when caliper try to instantiate it, error appears like this.

`hyper@hyperledger001:/opt/gopath/src/github.com/hyperledger/caliper$ node benchmark/simple/main.js -c config.json -n fabric.json
TAP version 13
# #######Caliper Test######
# Sleep 5s......
# Join channel......
# install all chaincodes......
info: [packager/Golang.js]: packaging GOLANG from contract/fabric/simple/go
info: [packager/Golang.js]: packaging GOLANG from contract/fabric/simple/go
ok 1 Installed chaincode simple successfully in all peers
# Instantiate chaincode......
not ok 2 Failed to instantiate chaincodes, Error: Failed to send instantiate due to error: Error: Failed to send instantiate Proposal or receive valid response. Response null or status is not 200. exiting... at Client.newDefaultKeyValueStore.then.then.then.then (/opt/gopath/src/github.com/hyperledger/caliper/src/fabric/e2eUtils.js:372:19) at <anonymous> at Client.newDefaultKeyValueStore.then.then.then.then.then (/opt/gopath/src/github.com/hyperledger/caliper/src/fabric/e2eUtils.js:385:15) at <anonymous>
  ---
    operator: fail
    at: chaincodes.reduce.then.catch (/opt/gopath/src/github.com/hyperledger/caliper/src/fabric/instantiate-chaincode.js:60:19)
    stack: |-
      Error: Failed to instantiate chaincodes, Error: Failed to send instantiate due to error: Error: Failed to send instantiate Proposal or receive valid response. Response null or status is not 200. exiting...
          at Client.newDefaultKeyValueStore.then.then.then.then (/opt/gopath/src/github.com/hyperledger/caliper/src/fabric/e2eUtils.js:372:19)
          at <anonymous>
          at Client.newDefaultKeyValueStore.then.then.then.then.then (/opt/gopath/src/github.com/hyperledger/caliper/src/fabric/e2eUtils.js:385:15)
          at <anonymous>
          at Test.assert [as _assert] (/opt/gopath/src/github.com/hyperledger/caliper/node_modules/tape/lib/test.js:224:54)
          at Test.bound [as _assert] (/opt/gopath/src/github.com/hyperledger/caliper/node_modules/tape/lib/test.js:76:32)
          at Test.fail (/opt/gopath/src/github.com/hyperledger/caliper/node_modules/tape/lib/test.js:317:10)
          at Test.bound [as fail] (/opt/gopath/src/github.com/hyperledger/caliper/node_modules/tape/lib/test.js:76:32)
          at chaincodes.reduce.then.catch (/opt/gopath/src/github.com/hyperledger/caliper/src/fabric/instantiate-chaincode.js:60:19)
          at <anonymous>
  ...
fabric.installSmartContract() failed, Error: Fabric: instantiate chaincodes failed
    at chaincodes.reduce.then.catch (/opt/gopath/src/github.com/hyperledger/caliper/src/fabric/instantiate-chaincode.js:61:31)
    at <anonymous>
[Transaction Info] - Submitted: 0 Succ: 0 Fail:0 Unfinished:0
unexpected error, Error: Fabric: instantiate chaincodes failed
    at chaincodes.reduce.then.catch (/opt/gopath/src/github.com/hyperledger/caliper/src/fabric/instantiate-chaincode.js:61:31)
    at <anonymous>

1..2
# tests 2
# pass  1
# fail  1
`

I saw my docker container by using cli, Installing chaincode is fine, but instantiated not completed.

**Comment by lotty02cho** [comment]
I realize that when I copy my chaincode using command, chaincode is not totally pasted well.

I figure out when I entered docker logs `dev-peer1.org1.honeybee.com-mycc-v1.0-....`

**Comment by lotty02cho** [comment]
when I enter my peer logs.

I saw that kinds of error, doe anyone knows about "lscc"?

`
2018-08-27 07:30:26.321 UTC [lscc] executeInstall -> INFO 30a3 Installed Chaincode [traceability] Version [v2.9] to peer
2018-08-27 07:30:26.385 UTC [endorser] ProcessProposal -> ERRO 30a4 [honeybeechannel][a9a602d9] simulateProposal() resulted in chaincode name:"lscc"  response status 500 for txid: a9a602d9988bb7ad23b1604dd66c299ddf24dc408b37036b91341264bcf76721
`

**Comment by lotty02cho** [comment]
when I restart my network, and start, there is some error in my docker chaincode.

`docker logs dev-peer0.org1.honeybee.com-traceability-v1.0

> traceability@0.0.0 start /usr/local/src
> node traceability.js "--peer.address" "peer0.org1.honeybee.com:7052"

E0827 08:26:26.412150365      20 ssl_transport_security.cc:238] Could not get common name of subject from certificate.
=========== Instantiated Chaincode ===========
hyper@hyperledger001:~$
hyper@hyperledger001:~$ > traceability@0.0.0 start /usr/local/src
The program 'start' is currently not installed. You can install it by typing:
sudo apt install upstart
hyper@hyperledger001:~$ > node traceability.js "--peer.address" "peer0.org1.honeybee.com:7052"
traceability.js: command not found
hyper@hyperledger001:~$
hyper@hyperledger001:~$ E0827 08:26:26.412150365      20 ssl_transport_security.cc:238] Could not get common name of subject from certificate.
E0827: command not found
hyper@hyperledger001:~$ =========== Instantiated Chaincode ===========
===========: command not found
`

**Comment by lotty02cho** [comment]
I also try to simpletest.go. but when chaincode is instantiated, it occurs like this.

 node benchmark/simple/main.js -c config.json -n fabric.json
TAP version 13
# #######Caliper Test######
# Sleep 5s......
# Join channel......
# install all chaincodes......
info: [packager/Node.js]: packaging Node from /opt/gopath/src/github.com/hyperledger/caliper/contract/fabric/simple/go
info: [packager/Node.js]: packaging Node from /opt/gopath/src/github.com/hyperledger/caliper/contract/fabric/simple/go
not ok 1 Failed to install chaincodes, Error: Failed to send install proposal due to error: Error: ENOENT: no such file or directory, scandir '/opt/gopath/src/github.com/hyperledger/caliper/contract/fabric/simple/go' at Client.newDefaultKeyValueStore.then.then.then (/opt/gopath/src/github.com/hyperledger/caliper/src/fabric/e2eUtils.js:148:15) at <anonymous>
  ---
    operator: fail
    at: chaincodes.reduce.then.catch (/opt/gopath/src/github.com/hyperledger/caliper/src/fabric/install-chaincode.js:65:19)
    stack: |-
      Error: Failed to install chaincodes, Error: Failed to send install proposal due to error: Error: ENOENT: no such file or directory, scandir '/opt/gopath/src/github.com/hyperledger/caliper/contract/fabric/simple/go'
          at Client.newDefaultKeyValueStore.then.then.then (/opt/gopath/src/github.com/hyperledger/caliper/src/fabric/e2eUtils.js:148:15)
          at <anonymous>
          at Test.assert [as _assert] (/opt/gopath/src/github.com/hyperledger/caliper/node_modules/tape/lib/test.js:224:54)
          at Test.bound [as _assert] (/opt/gopath/src/github.com/hyperledger/caliper/node_modules/tape/lib/test.js:76:32)
          at Test.fail (/opt/gopath/src/github.com/hyperledger/caliper/node_modules/tape/lib/test.js:317:10)
          at Test.bound [as fail] (/opt/gopath/src/github.com/hyperledger/caliper/node_modules/tape/lib/test.js:76:32)
          at chaincodes.reduce.then.catch (/opt/gopath/src/github.com/hyperledger/caliper/src/fabric/install-chaincode.js:65:19)
          at <anonymous>
  ...
fabric.installSmartContract() failed, Error: Failed to send install proposal due to error: Error: ENOENT: no such file or directory, scandir '/opt/gopath/src/github.com/hyperledger/caliper/contract/fabric/simple/go'
    at Client.newDefaultKeyValueStore.then.then.then (/opt/gopath/src/github.com/hyperledger/caliper/src/fabric/e2eUtils.js:148:15)
    at <anonymous>
[Transaction Info] - Submitted: 0 Succ: 0 Fail:0 Unfinished:0
unexpected error, Error: Failed to send install proposal due to error: Error: ENOENT: no such file or directory, scandir '/opt/gopath/src/github.com/hyperledger/caliper/contract/fabric/simple/go'
    at Client.newDefaultKeyValueStore.then.then.then (/opt/gopath/src/github.com/hyperledger/caliper/src/fabric/e2eUtils.js:148:15)
    at <anonymous>

1..1
# tests 1
# pass  0
# fail  1

hyper@hyperledger001:/opt/gopath/src/github.com/hyperledger/caliper$ vi benchmark/simple/fabric.json
hyper@hyperledger001:/opt/gopath/src/github.com/hyperledger/caliper$ node benchmark/simple/main.js -c config.json -n fabric.json
TAP version 13
# #######Caliper Test######
# Sleep 5s......
# Join channel......
# install all chaincodes......
info: [packager/Node.js]: packaging Node from /opt/gopath/src/github.com/hyperledger/caliper/src/contract/fabric/simple/go
info: [packager/Node.js]: packaging Node from /opt/gopath/src/github.com/hyperledger/caliper/src/contract/fabric/simple/go
ok 1 Installed chaincode simple successfully in all peers
# Instantiate chaincode......
error: [Peer.js]: sendProposal - timed out after:120000
error: [client-utils.js]: sendPeersProposal - Promise is rejected: Error: REQUEST_TIMEOUT
    at Timeout._onTimeout (/opt/gopath/src/github.com/hyperledger/caliper/node_modules/fabric-client/lib/Peer.js:124:19)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
error: [Peer.js]: sendProposal - timed out after:120000
error: [Peer.js]: sendProposal - timed out after:120000
error: [Peer.js]: sendProposal - timed out after:120000
error: [client-utils.js]: sendPeersProposal - Promise is rejected: Error: REQUEST_TIMEOUT
    at Timeout._onTimeout (/opt/gopath/src/github.com/hyperledger/caliper/node_modules/fabric-client/lib/Peer.js:124:19)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
error: [client-utils.js]: sendPeersProposal - Promise is rejected: Error: REQUEST_TIMEOUT
    at Timeout._onTimeout (/opt/gopath/src/github.com/hyperledger/caliper/node_modules/fabric-client/lib/Peer.js:124:19)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
error: [client-utils.js]: sendPeersProposal - Promise is rejected: Error: REQUEST_TIMEOUT
    at Timeout._onTimeout (/opt/gopath/src/github.com/hyperledger/caliper/node_modules/fabric-client/lib/Peer.js:124:19)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
not ok 2 Failed to instantiate chaincodes, Error: Failed to send instantiate due to error: Error: Failed to send instantiate Proposal or receive valid response. Response null or status is not 200. exiting... at Client.newDefaultKeyValueStore.then.then.then.then (/opt/gopath/src/github.com/hyperledger/caliper/src/fabric/e2eUtils.js:372:19) at <anonymous> at process._tickCallback (internal/process/next_tick.js:188:7) at Client.newDefaultKeyValueStore.then.then.then.then.then (/opt/gopath/src/github.com/hyperledger/caliper/src/fabric/e2eUtils.js:385:15) at <anonymous> at process._tickCallback (internal/process/next_tick.js:188:7)
  ---
    operator: fail
    at: chaincodes.reduce.then.catch (/opt/gopath/src/github.com/hyperledger/caliper/src/fabric/instantiate-chaincode.js:60:19)
    stack: |-
      Error: Failed to instantiate chaincodes, Error: Failed to send instantiate due to error: Error: Failed to send instantiate Proposal or receive valid response. Response null or status is not 200. exiting...
          at Client.newDefaultKeyValueStore.then.then.then.then (/opt/gopath/src/github.com/hyperledger/caliper/src/fabric/e2eUtils.js:372:19)
          at <anonymous>
          at process._tickCallback (internal/process/next_tick.js:188:7)
          at Client.newDefaultKeyValueStore.then.then.then.then.then (/opt/gopath/src/github.com/hyperledger/caliper/src/fabric/e2eUtils.js:385:15)
          at <anonymous>
          at process._tickCallback (internal/process/next_tick.js:188:7)
          at Test.assert [as _assert] (/opt/gopath/src/github.com/hyperledger/caliper/node_modules/tape/lib/test.js:224:54)
          at Test.bound [as _assert] (/opt/gopath/src/github.com/hyperledger/caliper/node_modules/tape/lib/test.js:76:32)
          at Test.fail (/opt/gopath/src/github.com/hyperledger/caliper/node_modules/tape/lib/test.js:317:10)
          at Test.bound [as fail] (/opt/gopath/src/github.com/hyperledger/caliper/node_modules/tape/lib/test.js:76:32)
          at chaincodes.reduce.then.catch (/opt/gopath/src/github.com/hyperledger/caliper/src/fabric/instantiate-chaincode.js:60:19)
          at <anonymous>
          at process._tickCallback (internal/process/next_tick.js:188:7)
  ...
fabric.installSmartContract() failed, Error: Fabric: instantiate chaincodes failed
    at chaincodes.reduce.then.catch (/opt/gopath/src/github.com/hyperledger/caliper/src/fabric/instantiate-chaincode.js:61:31)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
[Transaction Info] - Submitted: 0 Succ: 0 Fail:0 Unfinished:0
unexpected error, Error: Fabric: instantiate chaincodes failed
    at chaincodes.reduce.then.catch (/opt/gopath/src/github.com/hyperledger/caliper/src/fabric/instantiate-chaincode.js:61:31)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)

**Comment by lotty02cho** [comment]
I think it might be cause chaincode id, but I don't know why.
chaincodes.id should be changed in fabric.json.

I remember when I test caliper around 2 weeks ago, I just changed my chaincodes.version, but when I cloned this project yesterday, it doesn't work.

`"chaincodes":` [{"id": "traceability", "path": "src/contract/fabric/traceability/node", "language":"node", "version": "v1.0", "channel": `"honeybeechannel"}],`

after I changed my chaincodes.id like below, it works.
`"chaincodes":` [{"id": "traceability1_3", "path": "src/contract/fabric/traceability/node", "language":"node", "version": "v1.3", "channel": `"honeybeechannel"}],`
